src/components/Directives/index.js

```diff
@@ -71,6 +71,13 @@ const button = css`
 	box-shadow: 0px 10px 15px rgba(0, 0, 0, 0.1);
 `;
 
+const PLAY_LISTS = {
+	SEARCH_APP: 'search-app',
+	RELEVANT_SEARCH: 'relevant-search',
+	SEARCH_ANALYTICS: 'search-analytics',
+	ACCESS_CONTROL: 'access-control',
+};
+
```

Comment:
This helper should use const
https://github.com/appbaseio/dashboard/pull/429/files#diff-2797a3aa5ae22c0d11a52664c14ce8bd96e754271c4a8f2946ff328bdd29ae6fR96

This helper should use const
https://github.com/appbaseio/dashboard/pull/429/files#diff-2797a3aa5ae22c0d11a52664c14ce8bd96e754271c4a8f2946ff328bdd29ae6fR119

example:
```
switch(name)
   case: PLAYLIST.SEARCH_APP:
```

This functions should also use const